.ci/preInstall.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+# download the HERE SDK
+wget "$HERE_SDK_URL" -q -O HERE_SDK.zip
+unzip -j -d 'HERE_SDK' -o HERE_SDK.zip
+unzip -j "HERE_SDK/HERE-sdk.zip" "HERE-sdk/HERE-sdk/libs/HERE-sdk.aar" -d "HERE_SDK"
+rm HERE_SDK.zip
+
+# Find paths that contain an app module
+APP_PROJECTS=$(find $PWD -maxdepth 1 -type d -exec [ -x {}/app ] \; -print -prune)
```

Comment:
It's maybe a bit awkward (though not wrong per se) to check the directory to be executable. But in fact we are more interested in the `libs` directory here. How about changing the check to `[ -d {}/app/libs ]`?